src/test/java/org/medicmobile/webapp/mobile/AppUrlVerifierTest.java

```diff
@@ -24,13 +24,11 @@
 import org.mockito.junit.MockitoRule;
 import org.mockito.quality.Strictness;
 import org.robolectric.RobolectricTestRunner;
-import org.robolectric.annotation.Config;
 
 import java.io.IOException;
 import java.net.MalformedURLException;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(sdk = 28)
```

Comment:
Finally updated enough dependencies that we don't need to pin this back to 28.  :sweat_smile:   Now, the only tests that need the sdk version pinned are those that are testing functionality specific to an older Android version.